app/routes/routes.py

```diff
@@ -0,0 +1,266 @@
+    #print(get_cities())
+    # x = country_name_to_code("Poland")
+    # print(x)
+    # print(get_cities(x))
+    print("Warsaw")
+    if current_user.is_authenticated:
+        return redirect(url_for("user_homepage"))
+    else:
+        return render_template("main/start-page.html")
+
+
+@app.route("/lore")
+def lore():
+    return render_template("main/lore.html")
+
+
+@app.route("/register", methods=["GET", "POST"])
+def register():
+    form = RegistrationForm()
+    if form.validate_on_submit():
+        if not user_in_database(form.password.data, form.username.data):
+            user_password = form.password.data
+            site_session["username"] = form.username.data
+            site_session["email"] = form.email.data
+            site_session["password"] = form.password.data #cipher_suite.encrypt(user_password.encode())
+            return redirect(url_for("user_profile"))
+        else:
+            message = "Przykro mi ale masz nieprawidłowe dane"
+            return render_template(
+                "auth/register_form.html", form=form, register_failure_message=message
+            )
+    return render_template("auth/register_form.html", form=form)
+
+
+@app.route("/login", methods=["GET", "POST"])
+def login():
+    form = LoginForm()
+    if form.validate_on_submit():
+        if user_in_database(form.password.data) and if_user_is_admin(
+            form.name.data, form.password.data
+        ):
+            user = get_user_from_User_table(form.name.data)
+            login_user(user)
+            print(current_user.name)
+            return redirect(url_for("user_homepage"))
+
+        elif if_user_is_admin(form.name.data, form.password.data):
+            return redirect(url_for("admin_site"))
+
+        else:
+            return render_template(
+                "login.html", form=form, login_failure_message="Użytkownik nie istnieje"
+            )
+
+    elif not form.validate_on_submit() and form.submit_field.data:
+        failure_message = "Przykro mi ale nie udało ci się zalogować, sprawdź proszę dane które wprowadziłeś "
+        return render_template(
+            "login.html", form=form, login_failure_message=failure_message
+        )
+
+    else:
+        return render_template("login.html", form=form)
+
+
+@app.route("/FAQ")
+def FAQ_site():
+    return render_template("FAQ.html")
+
+
+@app.route("/user_profile", methods=["GET", "POST"])
+def user_profile():
+    form = UserProfileForm()
+    if form.validate_on_submit():
+        site_session["age"] = form.age.data
+        site_session["name"] = form.name.data
+        site_session["lastname"] = form.lastname.data
+        site_session["gender"] = form.gender.data
+        site_session["profile_description"] = form.profile_description.data
+        site_session["domicile"] = form.domicile.data
+        site_session["education"] = form.education.data
+        site_session["employment_status"] = form.employment_status.data

```

Comment:
storing these in the session is not a good idea :/. I would go for either creating a partial user item in the database, merging it with the route for uploading images or using js for multiple steps effect